### PR TITLE
py-copy-file

### PR DESCRIPTION
**Pull request by muriakula**

### CODE REVIEW

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
invalid input format
do it as u see in example
image
https://github.com/mate-academy/py-copy-file#:~:text=copy_file(%22cp%20file.txt%20file.txt%22)%20%20%23%20Does%20nothing

File: app/main.py [hunk 1]
```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_in = command.split(" ")[1]
+    file_out = command.split(" ")[2]
```
Comment (EdAlekseiev) [inline app/main.py:3]: Don't split `command` twice. Create a separate variable for storing result of `command.split()` and use it where you need.

Comment (EdAlekseiev) [inline app/main.py:3]: `" "` is default value `split`, you can use just `command.split()`

File: app/main.py [hunk 2]
```diff
@@ -1 +1,6 @@
...
+    if file_in != file_out:
```
Comment (EdAlekseiev) [inline app/main.py:4]: Also, check that command starts from `cp`.

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 3]
```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    elements = command.split()
```
Comment (olena-hul) [inline app/main.py:2]: Check command length, it should equal exactly to 3 arguments
Then use unpacking, for example:
a, b, c = [1,2,3]
This way you'll avoid accessing command items using indexes

**Review by olena-hul — CHANGES_REQUESTED** [review]

**Review by Bohdan-Salamakha — APPROVED** [review]

**Pull request closed without merging**